scripts/projectView.js

```diff
@@ -0,0 +1,84 @@
+// Configure a view object, to hold all our functions for dynamic updates and article-related event handlers.
+var portfolioPage = {};
+
+portfolioPage.populateFilter = function() {
+  $('article').each(function() {
+    if (!$(this).hasClass('template')) {
+
+      var val = $(this).attr('data-category');
+      var optionTag = '<option value="' + val + '">' + val + '</option>';
+      if ($('#category-filter option[value="' + val + '"]').length === 0) {
+        $('#category-filter').append(optionTag);
+      }
+    }
+  });
+};
+
+portfolioPage.handleCategoryFilter = function() {
+  $('#category-filter').on('change', function() {
+    if ($(this).val()) {
+      $('article').hide();
+      var val = $(this).val();
+      $('article').each(function() {
+        if ($(this).attr('data-category') === val) {
+          $(this).fadeIn(1000);
+        }
+      });
+
+    } else {
+
+      $('article').show();
+      $('article.template').hide();
+
+    }
+  });
+
+};
+
+portfolioPage.handleMainNav = function() {
+  $('.main-nav').on('click', '.tab', function() {
+    var val = $(this).attr('data-content');
+    console.log(val);
+    $('.tab-content').hide();
+    $('.tab-content').each(function(index) {
+      if ($(this).attr('id') === val) {
+        $(this).fadeIn(2000);
+      }
+    });
+  });
+  $('.main-nav .tab:first').click();
+};
+
+portfolioPage.setTeasers = function() {
+    // Hide any elements after the first 2 (<p> tags in this case)
```

Comment:
Don't just copy-paste from the pair assignment, that doesn't help your learning nearly as much as even retyping it would.